modules/auxiliary/server/capture/smb.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'ruby_smb'
 require 'ruby_smb/gss/provider/ntlm'
-require 'metasploit/framework/hashes/identify'
+require 'metasploit/framework/hashes'
```

Comment:
This module should not need this require.
```suggestion
require 'metasploit/framework/hashes'
```